### PR TITLE
Fix argmax_channel method and add preqs in scala mnist tutorial (#7101)

### PR DESCRIPTION
**Pull request by kottmann**

### CODE REVIEW

File: docs/tutorials/scala/mnist.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ This Scala tutorial guides you through a classic computer vision application: id
 
 Let's train a 3-layer network (i.e multilayer perceptron network) on the MNIST dataset to classify handwritten digits.
 
+## Prerequisites
+To complete this tutorial, we need:
+
+- to compile the latest MXnet version. See MXnet the installation instructions for your operating system in [Setup and Installation](http://mxnet.io/get_started/install.html).
```
Comment (madjam) [inline docs/tutorials/scala/mnist.md]: Mxnet -> MXNet
See Mxnet the installation -> See the MXNet installation

Comment (kottmann) [inline docs/tutorials/scala/mnist.md]: Sorry, corrected it

**Comment by piiswrong** [comment]
@madjam @javelinjs @Ldpe2G 
Are you sure this is correct?
The scala package should convert names to camel cases I think?

**Comment by Ldpe2G** [comment]
@piiswrong  we did not convert the names to camel cases and just use  original names.

**Pull request merged**